routers/bookings.py

```diff
@@ -0,0 +1,96 @@
+from fastapi import APIRouter
+
+router = APIRouter()
+
+
+@router.get("/booking")
+async def bookings_get():
+    """
+    List bookings
+
+        Args:
+            check_in (date)
+            check_out (date)
+            is_paid (bool)
+            is_active (bool)
+        Returns:
+            JSON with result
+    """
+    pass
+
+
+@router.post("/booking")
+async def booking_create():
+    """
+    Add booking
+
+        Args:
+            room_number (int): number of the room
+            guest_id (int): guest who is booking
+            check_in (date): when move in
+            check_out (date): when move out
+            description (str): desription for the booking
+        Returns:
+            JSON with result
+    """
+    pass
+
+
+@router.post("/booking/{booking_id}")
+async def bookings_update():
+    """
+    Update booking
+
+        Args:
+            bookig_id (int): id of the booking
+            room_number (int): number of the room
+            guest_id (int): guest who is booking
+            check_in (date): when move in
+            check_out (date): when move out
+            description (str): desription for the booking
+        Returns:
+            JSON with result
+    """
+    pass
+
+
+@router.post("/booking/{booking_id}/active")
+async def booking_set_active(is_active: bool):
+    """
+    Set booking state
+
+        Args:
+            booking_id (int): id of the booking
+            is_active (bool): status of the booking
+        Returns:
+            JSON with result
+    """
+    pass
+
+
+@router.post("/booking/{booking_id}/review")
+async def booking_post_review():
+    """
+    Update client's review
+
+        Args:
+            booking_id (int): id of the booking
+            is_active (bool): status of the booking
+        Returns:
+            JSON with result
+    """
+    pass
+
+
+@router.get("/booking/{booking_id}/payment")
+async def booking_get_payment():
+    """
+    Update client's review
```

Comment:
Wrong docstring
Also why do you need to get payment separately?

Reply:
> ....why do you need to get payment separately?
Only for the convenience of the user. It was planned use in crud only one function for requests of the same type from different endpoints

Reply:
do not create these duplicates, it could be confusing to users, it brings no value, and if you change that function you'll need to update 2 endpoints.

Reply:
Done with direct duplicate.  What about indirect? As example, there is no direct connection between room_number and requests. Can I use endpoints for receiving requests based on room number?